model.go

```diff
@@ -0,0 +1,25 @@
+/*
+This file will contain the data models. Until refactored only Headers will be here to be included everywhere else.
+*/
+
+package profitbricks
+
+import "net/http"
+
+type Header struct {
```

Comment:
What's the purpose of this struct, when you have `BaseResource`?

Reply:
BaseResource was a left over from previous refactor PR. Is now removed